docs/root/configuration/advanced/well_known_filter_state.rst

```diff
@@ -13,9 +13,16 @@ The following list of filter state objects are consumed by Envoy extensions:
    * - **Filter state key**
      - **Purpose**
    * - ``envoy.tcp_proxy.cluster``
-     - :ref:`TCP proxy <config_network_filters_tcp_proxy>` dynamic cluster selection on a per-connection basis.
+     - | :ref:`TCP proxy <config_network_filters_tcp_proxy>` dynamic cluster name selection
+       | on a per-connection basis.
    * - ``envoy.network.transport_socket.original_dst_address``
      - | :ref:`Original destination cluster <arch_overview_load_balancing_types_original_destination>` dynamic address selection.
        | Fields:
        | - *ip*: IP address value;
        | - *port*: port value.
+   * - ``envoy.upstream.dynamic_host``
```

Comment:
@phlax this sufficient for docs or do we wantan example se case or example yaml?

Reply:
im not sure i understand the change exactly - not sure that means it needs more docs or just that i need more context

Reply:
This is aimed at:
1) developers (advanced topic) so they can write their filters to influence other filters via the filter state.
2) telemetry config, so they know which fields to use to unpack the filter state.

Let me know how I can make it better!

Reply:
k  - kinda following - so i guess any example/configs etc would be on the telemetry docs themselves

i guess my question would be are there any such examples that use these fields - example bootstrap configs can be v helpful, but also benefit from getting tested

Reply:
Added a format string as an example.